manifests/params.pp

```diff
@@ -227,6 +226,11 @@
   $tftp_listen_on         = 'https'
   $tftp_managed           = true
   $tftp_manage_wget       = true
+  if $tftp {
+    $tftp_root = lookup('tftp::root')
+  } else {
+    $tftp_root = '/not/used'
+  }
```

Comment:
This is params.pp so `$tftp` is static. This means it will never be looked up and always `/not/used`.

Reply:
Ah... right.... 
I'll try to look for another solution.

Reply:
You could provide a default value to `lookup`.

Reply:
https://github.com/theforeman/puppet-foreman_proxy/pull/673 :-D